controllers/authorino_controller.go

```diff
@@ -191,8 +192,15 @@ func (r *AuthorinoReconciler) buildAuthorinoDeployment(authorino *api.Authorino)
 	var containers []k8score.Container
 	var saName = authorino.Name + "-authorino"
 
-	if authorino.Spec.Image == "" {
-		authorino.Spec.Image = DefaultAuthorinoImage
+	image := authorino.Spec.Image
+
+	if image == "" {
+		image = env.GetString("RELATED_IMAGE_AUTHORINO", DefaultAuthorinoImage)
+	}
+
+	if image == "" {
+		// `DefaultAuthorinoImage can be empty string. But image cannot be or deployment will fail
+		image = "quay.io/kuadrant/authorino:latest"
```

Comment:
I guess we can't set the default value of `DefaultAuthorinoImage` to the image because it's not unset it's empty string in this case?

Reply:
exactly